packages/web-config-server/src/apiV1/dataBuilders/DataBuilder.js

```diff
@@ -46,7 +46,7 @@ export class DataBuilder {
     dataElementCodes,
     additionalQueryConfig,
     aggregationType = this.aggregationType,
-    aggregationConfig = {},
+    aggregationConfig = this.config.aggregationConfig,
```

Comment:
```suggestion
    aggregationConfig = this.config.aggregationConfig || {},
```
`aggregationConfig` might not be defined within `config` right?